includes/class-amp-cli.php

```diff
@@ -263,6 +263,69 @@ public function validate_site( $args, $assoc_args ) {
 		WP_CLI::line( sprintf( __( 'For more details, please see: %s', 'amp' ), $url_more_details ) );
 	}
 
+	/**
+	 * Reset all validation data on a site.
+	 *
+	 * This deletes all amp_invalid_url posts and all amp_validation_error terms.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--yes]
+	 * : Proceed to empty the site validation data without a confirmation prompt.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp amp reset-site-validation --yes
+	 *
+	 * @subcommand reset-site-validation
+	 * @param array $args       Positional args. Unused.
+	 * @param array $assoc_args Associative args.
+	 * @throws Exception If an error happens.
+	 */
+	public function reset_site_validation( $args, $assoc_args ) {
```

Comment:
Great idea, this is something I've had to do manually with longer WP-CLI commands to delete the error posts and terms.

Reply:
This worked well:

<img width="473" alt="reset-site-validation" src="https://user-images.githubusercontent.com/4063887/45889578-3283a600-bd97-11e8-84a4-f76827b520c0.png">

<img width="937" alt="no-invalid-urls" src="https://user-images.githubusercontent.com/4063887/45889630-4fb87480-bd97-11e8-9166-290f81dfdae9.png">